scipy/integrate/_quadpack_py.py

```diff
@@ -920,7 +921,12 @@ def ranges1(*args):
             opts={"epsabs": epsabs, "epsrel": epsrel})
 
 
-def nquad(func, ranges, args=None, opts=None, full_output=False):
+QuadratureResult = _make_tuple_bunch("QuadratureResult",
+                                     field_names=["integral", "abserr"],
+                                     extra_field_names=["neval"])
```

Comment:
Just a note. Now that you are using PyCharm, it would be nice if we find a way to make this work with autocompletion. Not fully sure why PyCharm does not like this, but it prevents that vs a normal (data)class.